adding some data to a new incentive data folder with descriptions

anupama-sitaraman opened the pull request.

VisionistJavascript committed:
NEAT

anupama-sitaraman committed:
adding some data

anupama-sitaraman committed:
Merge branch 'main' into incentives

coopersigrist commented on Data/Incentives/census_by_zip_complex.csv:
I added Hispanic population (and Hispanic_prop) to the census data, you should add that here -- note that each of the categories from the dataset are people who identify with at least the race given, but also could identify as another, these numbers add to higher than the total pop, I need to add a note about that in details.md of Census -- if you write yours you should add it there too I wont touch it for a bit

coopersigrist commented on Data/Incentives/README.md:
small thing, I've been naming all my README.md into details.md and the only README.md is going to be the one for the main page of the repo -- do you know a reason this would matter? If not then I prefer this way

anupama-sitaraman replied:
nope, just mistakenly did README. will change this!

coopersigrist commented on Data/Incentives/EIA_Annual_Household_Site_Fuel_Consumption.csv:
is this the most recent available data?

anupama-sitaraman replied:
yes - we only have 2020 data since this data is collected roughly every 5 years, and I haven't seen the info for 2025 yet (probably still underway). I'm assuming that the kwh used annually per home is roughly the same in 2022 as it was in 2020 for the modeling.

coopersigrist commented on Data/Incentives/README.md:
Based on my other comment you'll need to remove census and EIA from this list

coopersigrist commented on Data/Incentives/EIA_Annual_Household_Site_Fuel_Consumption.csv:
Please move this to DATA/EIA and add a description of this data to the details.md

coopersigrist commented on Data/Incentives/census_by_zip_complex.csv:
Move this to Data/Census -- and move the comment from the readme as well to details.md

Replace the current census_by_zip.csv (combine if there are things in the original such as hispanic pop)

coopersigrist commented on Data/Incentives/elec_rate_zipcodes_2022.csv:
I'd like this in Data/Grid (may be renamed) -- but I if you feel strongly about it being here I'm willing to let it stay

coopersigrist commented on Data/Incentives/CSI_training_data.csv:
Move to a new folder Data/Training/ -- add a detail.md that explains it

coopersigrist commented:
mostly I would just like these datasets moved into the categories I already have and the details.md to be updated. 

You can check out Data/data_load_util.py to see how I load the datasets, it shouldn't be any harder to use them spread out than all in this folder and that's the point of the organization I used.

coopersigrist reviewed (changes requested):
mostly I would just like these datasets moved into the categories I already have and the details.md to be updated.

You can check out Data/data_load_util.py to see how I load the datasets, it shouldn't be any harder to use them spread out than all in this folder and that's the point of the organization I used.

anupama-sitaraman commented:
> mostly I would just like these datasets moved into the categories I already have and the details.md to be updated.
> 
> You can check out Data/data_load_util.py to see how I load the datasets, it shouldn't be any harder to use them spread out than all in this folder and that's the point of the organization I used.

gotcha - I wasn't sure how separated you'd want these to be, so I ended up separating everything from your code so there was no merge issues. I'll make these fixes and reintegrate everything!

anupama-sitaraman committed:
Merge branch 'main' into incentives

anupama-sitaraman committed:
Merge branch 'incentives' of https://github.com/coopersigrist/SunSight into incentives

anupama-sitaraman committed:
mod to scrape util (adding region mappings, adding zip to region), adding more to scrape

anupama-sitaraman committed:
update data

anupama-sitaraman committed:
dump of everything needed to run incentives (should be refactored later)

forgot to dump

remove model training code

forgotten commits
dumped all incentive code and sqaushed smaller commits

anupama-sitaraman committed:
dump of everything needed to run incentives (should be refactored later)

anupama-sitaraman committed:
forgot to dump

anupama-sitaraman committed:
remove model training code

anupama-sitaraman committed:
forgotten commits

anupama-sitaraman committed:
Merge branch 'incentives' of https://github.com/coopersigrist/SunSight into incentives

anupama-sitaraman committed:
redid incentives by state

anupama-sitaraman committed:
latest updated state-level payback results
